test/client/script.js

```diff
@@ -3,9 +3,17 @@
 import { base64encode } from 'belter/src';
 
 import { getClientID, getIntent, getCurrency, getVault, getCommit, getClientToken, getPartnerAttributionID,
-    getMerchantID, getClientAccessToken, getSDKIntegrationSource, insertMockSDKScript, getPageType } from '../../src';
+    getMerchantID, getClientAccessToken, getSDKIntegrationSource, insertMockSDKScript, getPageType, getLocale } from '../../src';
 
 describe(`script cases`, () => {
+    beforeEach(() => {
+        /* eslint-disable compat/compat */
```

Comment:
@amyegan what do you think about using `/* eslint-disable compat/compat */` for the whole test file? My initial thought is none of the test files should care about the `compat` rules.